### ISSUE TITLE
'v-model' directives aren't supported on <Select> elements.

### ISSUE THREAD

**Issue by vvpvvp** [vuejs/vetur#303]
## Info

Platform: macOS
Vetur version: 0.8.6
VSCode version: 1.13.1

## Problem

'v-model' directives aren't supported on \<Select\> elements.

\<Select\> is a custom component.

<img width="1002" alt="2017-06-30 11 41 10" src="https://user-images.githubusercontent.com/7303371/27720075-db1f0102-5d89-11e7-8d0b-78d808927da4.png">

**Comment by HerringtonDarkholme** [comment]
My suggestion is using another name for custom component.  Case sensitivity is only supported in SFC and not html compliant.

@mysticatea however, eslint-plugin-vue should be either consistently case insensitive or sensitive, not sometimes.

**Comment by vvpvvp** [comment]
Accordding the [Vue Component-Naming-Conventions](https://vuejs.org/v2/guide/components.html#Component-Naming-Conventions)，We can use pascalCased component in template, and \<Select\> is different to \<select\>.

**Comment by HerringtonDarkholme** [comment]
**can** doesn't imply **should**. The doc also explicitly states:

> This means that the PascalCase is the most universal declaration convention and kebab-case is the most universal usage convention.

**Comment by vvpvvp** [comment]
but I think, It's doesn't imply should not.
It's vue template, and It's virtual computing, case sensitive doesn't imply wrong, and It's supported.

**Comment by HerringtonDarkholme** [comment]
Try this:

```ts
new Vue({template: '<Div>I'm div, case insensitive</Div>'}).$mount()
```

```ts
new Vue({template: '<NoExist>Error</NoExist>'}).$mount()
```

Vue's string template, and SFC is case insensitive for html compliance. It seems hard to correctly tell a custom component from a misspell native tag name without resolving the `components` field of component associated with the template.

**Comment by vvpvvp** [comment]
I think you are consider the wrong direction, 
if you say we should case insensitive,  regardless of whether my component is custom, it's just a \<select\> tag,  it should not be wrong.

**Comment by HerringtonDarkholme** [comment]
Please file this issue on https://github.com/vuejs/eslint-plugin-vue .

**Comment by armano2** [comment]
@octref this issue is was solved some time ago in vue-eslint-parser/plugin and you should consider updating version up to 3.13.0 from currently used 3.8.0

cc: @thearabbit, @HerringtonDarkholme , @vvpvvp

**Comment by HerringtonDarkholme** [comment]
@vvpvvp @armano2 We already upgraded to  3.13.0

https://github.com/vuejs/vetur/blob/master/server/package.json#L21

The fix will be released in next version, soon.

**Comment by octref** [comment]
Released 0.9.6.
